hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/transaction/properties/SimulatedBlockMetaSource.java

```diff
@@ -0,0 +1,17 @@
+package com.hedera.mirror.web3.transaction.properties;
```

Comment:
We intend to have future REST APIs in this module so let's move all packages in this PR into `com.hedera.mirror.web3.evm` package.

```suggestion
package com.hedera.mirror.web3.evm.transaction.properties;
```